bioconda_utils/cran_skeleton.py

```diff
@@ -24,20 +24,19 @@
 # Raw strings needed to support the awkward backslashes needed when adding the
 # command to yaml
 gpl2_short = r"  license_family: GPL2"
-gpl2_long = (
-    "  license_family: GPL2\n  license_file: '{{ environ[\"PREFIX\"] }}"
-    "\/lib\/R\/share\/licenses\/GPL-2'  # [unix]\n  "
-    "license_file: '{{ environ[\"PREFIX\"] }}"
-    "\\\R\\\share\\\licenses\\\GPL-2'  # [win]"
-)
+gpl2_long = r"""
+  license_family: GPL2
+  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'  # [unix]
+  license_file: '{{ environ["PREFIX"] }}\\R\\share\\licenses\\GPL-2'  # [win]
+""".strip('\n')
 
 gpl3_short = r"  license_family: GPL3"
-gpl3_long = (
-    "  license_family: GPL3\n  license_file: '{{ environ[\"PREFIX\"] }}"
-    "\/lib\/R\/share\/licenses\/GPL-3'  # [unix]\n  "
-    "license_file: '{{ environ[\"PREFIX\"] }}"
-    "\\\R\\\share\\\licenses\\\GPL-3'  # [win]"
-)
+gpl3_long = r"""
+  license_family: GPL3
+  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'  # [unix]
+  license_file: '{{ environ["PREFIX"] }}\\R\\share\\licenses\\GPL-3'  # [win]
```

Comment:
These strings are later used in `filter_lines_regex` as the `substitute`s for `re.sub(regex, substitute, line)` which means we need the double `\` because `re.sub` processes backslash-escapes in those strings.

Reply:
Ah, the docs say https://docs.python.org/3/library/re.html#re.sub:
>  If the pattern isn’t found, string is returned unchanged. repl can be a string or a function; if it is a string, any backslash escapes in it are processed.

So, we could also make this
```diff
-gpl3_long = r"""
+def gpl3_long():
+    return r"""
```
if we would want to avoid that strange subtlety.